### ISSUE TITLE
Be clean consistent about .gitignore files

### ISSUE THREAD

**Issue by sagikazarmark** [fuelphp/boilerplate#1]
Do we have to provide an entry for all kinds of cache and IDE? We should keep it simple:

```
bin/
tests/_output/
vendor/
codeception.yml
composer.lock
```

**Comment by stevewest** [comment]
I would rather that ide files and where kept as not everyone knows about or uses global git ignores.

**Comment by sagikazarmark** [comment]
What about Thumbs.db and other files?

**Comment by stevewest** [comment]
I think they should be included too, I'd rather be overcautious about the gitignore as imo it does not matter if it's tidy or not as it's not something people access every day themselves.

**Comment by sagikazarmark** [comment]
Okay. I am going to tidy it up a bit: group IDE/cache files, directories, etc.

**Comment by stevewest** [comment]
Sure

**Comment by sagikazarmark** [comment]
Ok, so I ended up with [this](https://github.com/fuelphp/boilerplate/commit/2347b4324abd5c00c3bf8181b99a437ee98d922c?diff=split). Any thoughts?

**Comment by sagikazarmark** [comment]
Actually, I am still not sure that all this IDEs, and other things should be included in the .gitignore. You can't ignore ALL IDE files in the world, so based on what do we ignore these? Same for cache files.

**Comment by stevewest** [comment]
No, we can't. But I'd say do the best we can as I don't trust others to be using their global gitignores.
